Source/WebKit/UIProcess/Cocoa/XPCConnectionTerminationWatchdog.mm

```diff
@@ -41,13 +45,19 @@
     : m_xpcConnection(process.connection()->xpcConnection())
     , m_watchdogTimer(RunLoop::main(), this, &XPCConnectionTerminationWatchdog::watchdogTimerFired)
     , m_assertion(ProcessAndUIAssertion::create(process, "XPCConnectionTerminationWatchdog"_s, ProcessAssertionType::Background))
+    , m_process(WeakPtr { process })
 {
     m_watchdogTimer.startOneShot(interval);
 }
     
 void XPCConnectionTerminationWatchdog::watchdogTimerFired()
 {
     terminateWithReason(m_xpcConnection.get(), ReasonCode::WatchdogTimerFired, "XPCConnectionTerminationWatchdog::watchdogTimerFired");
```

Comment:
This doesn't work when using EXTENSIONKIT, so we should only call it in the `#else` case.